vassal-app/src/test/resources/clirr-ignored-differences.xml

```diff
@@ -1265,4 +1265,15 @@
         <differenceType>6008</differenceType>
         <field>pieceMovers</field>
     </difference>
+    <difference>
+        <className>VASSAL/build/module/map/CounterDetailViewer</className>
+        <differenceType>6007</differenceType>
+        <field>showDeckDepth</field>
+    </difference>
+    <difference>
+        <className>VASSAL/build/module/map/CounterDetailViewer$Visitor</className>
+        <differenceType>7004</differenceType>
+        <method>**</method>
+        <justification>Nothing external should use this</justification>
+    </difference>
```

Comment:
I should run the dependency checker to see if this will blow up any modules. (I suspect it could blow up VASL.)

Reply:
Yeah worth checking. I could make a parallel one if so.

Reply:
This will break VASL, as it has a class extending CounterDetailViewer.